components/ai_chat/resources/untrusted_conversation_frame/components/conversation_entries/index.tsx

```diff
@@ -77,6 +86,7 @@ function ConversationEntries() {
             ? getCompletion(latestTurn)
             : latestTurn.text
           const lastEditedTime = latestTurn.createdTime
+          const hasReasoning = latestTurnText.includes('<think>')
```

Comment:
Hey, so here we hardcode `think` as the tag, but above its configurable - maybe we just hardcode think, until it changes?

Reply:
Changed to hardcoded `<think>` value